shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -121,10 +131,43 @@ - (void)userNotificationCenter:(UNUserNotificationCenter*)center
   }
 }
 
+static BOOL IsDeepLinkingEnabled() {
+  NSDictionary* infoDict = [[NSBundle mainBundle] infoDictionary];
+  NSNumber* isEnabled = [infoDict objectForKey:@"FlutterDeepLinkingEnabled"];
```

Comment:
The key name may be confusing. Developers can still do their own deeplinking if they write custom plugin. this is really for the default push route behavior.

@johnpryan any suggestion for the name?

Reply:
That's a lot to fit into a name, I think we'll have provide more context in our docs (either the API docs or on flutter.dev)

Reply:
@chunhtai sound good?  We keep it like this and define it in our docs?  I agree with john that it is impossible to convey all nuances in so few characters.

Reply:
yes that sounds good to me, I have updated the android part to match